content/reference/sql/version-control/dolt-sql-functions.md

```diff
@@ -221,6 +222,126 @@ The results from `DOLT_DIFF()` show how the data has changed going from `main` t
 +---------+-------+---------+----------+----------------+-----------------------------------+-----------+---------+---------------+-------------+-----------------------------------+-----------+
 ```
 
+## `DOLT_DIFF_SUMMARY()`
+
+The `DOLT_DIFF_SUMMARY()` table function calculates the data difference summary in a table's data at any two commits 
+in the database. Each row in the result set describes a diff summary for a single table with statistics information of
```

Comment:
In between these two sentences might be a good spot to reiterate that schema changes such as creating a new table with no rows, or deleting a table with no rows will not show up in these results. You did a good job mentioning that this is data differences only in the first sentence, but might be worth reiterating in case customers gloss over that important word in the first sentence.